app/util/autoversioning.py

```diff
@@ -58,18 +61,21 @@ def write_package_version_file(package_version_string):
 
 def restore_original_package_version_file():
     """
-    Restore the backed up version of package_version.py. This is just a convenience method to help us remember not to
-    commit changes to the package version file.
+    Restore the backed up version of package_version.py. This is just a convenience method to help
+    us remember not to commit changes to the package version file.
     """
-    _try_rename(_VERSION_FILE_BACKUP_PATH, _VERSION_FILE_PATH)
+    if not _try_rename(_VERSION_FILE_BACKUP_PATH, _VERSION_FILE_PATH):
```

Comment:
This is a workaround to prevent the frozen version from breaking unit test code.